packages/extension/package.json

```diff
@@ -3,78 +3,70 @@
   "version": "3.7.0",
   "description": "GemWallet browser extension",
   "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vite --host",
+    "build": "rm -rf dist && tsc && vite build -c vite.config.ts && vite build -c vite.content-script.config.ts",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "test": "vitest",
+    "cypress:e2e": "cypress open --browser chrome --e2e"
+  },
   "dependencies": {
-    "@craco/craco": "^6.4.0",
     "@emeraldpay/hashicon-react": "^0.5.2",
-    "@emotion/react": "^11.7.0",
-    "@emotion/styled": "^11.6.0",
-    "@mui/icons-material": "^5.14.13",
-    "@mui/material": "^5.14.13",
-    "@sentry/react": "^7.91.0",
-    "copy-to-clipboard": "^3.3.1",
-    "crypto-js": "^4.1.1",
-    "lottie-react": "^2.1.0",
-    "moment": "^2.29.4",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.15.5",
+    "@mui/material": "^5.15.5",
+    "@sentry/react": "^7.93.0",
+    "copy-to-clipboard": "^3.3.3",
+    "crypto-browserify": "^3.12.0",
+    "crypto-js": "^4.2.0",
+    "dayjs": "^1.11.10",
+    "events": "^3.3.0",
+    "lottie-react": "^2.4.0",
     "prismjs": "^1.29.0",
     "qrcode.react": "^3.1.0",
-    "react": "^17.0.2",
-    "react-content-loader": "^6.2.0",
-    "react-dom": "^17.0.2",
-    "react-icons": "^4.11.0",
+    "react": "^18.2.0",
+    "react-content-loader": "^6.2.1",
+    "react-dom": "^18.2.0",
+    "react-icons": "^5.0.1",
     "react-infinite-scroll-component": "^6.1.0",
     "react-json-view": "^1.21.3",
     "react-lazy-load-image-component": "^1.6.0",
-    "react-router-dom": "6",
-    "react-scripts": "4.0.3",
+    "react-router-dom": "^6.21.3",
     "react-simple-code-editor": "^0.13.1",
-    "ripple-keypairs": "^1.3.1",
-    "typescript": "^4.1.2",
-    "web-vitals": "^1.0.1",
-    "xrpl": "^2.14.0",
-    "xrpl-accountlib": "^3.2.6",
-    "xrpl-secret-numbers": "^0.3.4"
+    "stream-browserify": "^3.0.0",
+    "web-vitals": "^3.5.1",
+    "xrpl": "^3.0.0",
+    "xrpl-accountlib": "^3.2.9"
   },
   "devDependencies": {
     "@gemwallet/constants": "*",
     "@gemwallet/eslint-config": "*",
```

Comment:
You are pulling our eslint configuration, though we can see in the dev dependencies that you are using the dependencies from vite.js as you require another version of eslint.
We have two solutions here:

- We use the configuration from vite.js for the linter on the extension and the package @gemwallet/eslint-config for the API
- We update our configuration within @gemwallet/eslint-config to match vite and override it, and we make use of it within the extension.

We can also do this in another MR to don't add more change as this is not blocker.

What do you think?